README.md

```diff
@@ -10,6 +10,49 @@ intended for testing new features, and can deploy sandboxes automatically from
 GitHub pull requests.
 
 
+What does OpenCraft Instance Manager do and how to use it
+---------------------------------------------------------
+
+This section focuses on how to use the web interface, as opposed to how to install the code,
```

Comment:
*how to install, **debug, or develop Ocim; see the following sections for that**.*